db/schema.rb

```diff
@@ -833,6 +834,16 @@
     t.index ["account_uuid"], name: "tud_account_availability_logs"
   end
 
+  create_table "test_user_dashboard_tud_account_checkouts", force: :cascade do |t|
```

Comment:
Will it cause any problems to create this table from the schema if it's not being added in a migration in this PR? I.E. what happens if the migration that creates this table gets committed later, this table will already exist?

I don't see a migration with this table name in it, where did it come from?

Reply:
@acovrig thanks for catching this! @LindseySaari and I are going to work on it tomorrow.